script.js

```diff
@@ -4,3 +4,102 @@
 // - A User should only be able to select one box per turn
 // - A User should not be able to select a box that is already selected
 // - When game is over (whether a win or tie) the users should be prompted to play again and the game should be reset
+
+let gamePiece = "";
+let playerTurn = true;
+let text = document.getElementById("text");
+const xGamePiece = "X";
+const oGamePiece = "O";
+const gameSpaces = document.querySelectorAll(".gameSpace");
+
+gameSpaces.forEach(addEventListenerToGameSpaces);
+
+function addEventListenerToGameSpaces(gameSpace) {
+  gameSpace.addEventListener("click", useGamePiece);
+}
+
+text.innerHTML = "Player X turn";
+
+function checkWin() {
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[1].innerHTML &&
+    gameSpaces[1].innerHTML === gameSpaces[2].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+
+  if (
+    gameSpaces[3].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[5].innerHTML &&
+    gameSpaces[3].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[6].innerHTML === gameSpaces[7].innerHTML &&
+    gameSpaces[7].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[6].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[3].innerHTML &&
+    gameSpaces[3].innerHTML === gameSpaces[6].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[1].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[7].innerHTML &&
+    gameSpaces[1].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[2].innerHTML === gameSpaces[5].innerHTML &&
+    gameSpaces[5].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[2].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[2].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[6].innerHTML &&
+    gameSpaces[2].innerHTML !== ""
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function useGamePiece(event) {
+  if (playerTurn) {
+    gamePiece = xGamePiece;
+  }
+  if (playerTurn === false) {
```

Comment:
part of my previous comment this could just simply be an else
or you could use a ternary